### PR TITLE
PXB-2572: Archive results directory for debugging purposes for both PXB

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Builds: https://pxb.cd.percona.com/job/percona-xtrabackup-8.0-test-param-illiatests/13/

### CODE REVIEW

File: pxb/v2/jenkins/percona-xtrabackup-2.4-test-param.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@
     properties:
     - build-discarder:
         days-to-keep: -1
-        num-to-keep: 50
+        num-to-keep: 30
```
Comment (altmannmarcelo) [inline pxb/v2/jenkins/percona-xtrabackup-2.4-test-param.yml:14]: Is this the number of test params to keep or the number of test pipelines?

We have 8 OS that get tested against 4 different server, resulting in 32 test pipelines for each test param

Comment (Sudokamikaze) [reply pxb/v2/jenkins/percona-xtrabackup-2.4-test-param.yml:14]: This is only for param runs, not configurations and not of pipeline

Comment (altmannmarcelo) [reply pxb/v2/jenkins/percona-xtrabackup-2.4-test-param.yml:14]: ack

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**